script/sandbox-up.sh

```
@@ -71,14 +70,9 @@ for i in $(seq 0 $((NODE_COUNT - 1))); do
-# Export to the nextest test environment when invoked as a setup script.
-if [ -n "${NEXTEST_ENV:-}" ]; then
-  {
-    echo "NEAR_SANDBOX_RPC_URL=$(cat "$(addr_file 0)")"
-    echo "SANDBOX_NODE_COUNT=${NODE_COUNT}"
-    for i in $(seq 0 $((NODE_COUNT - 1))); do
-      echo "NEAR_SANDBOX_RPC_URL_${i}=$(cat "$(addr_file "$i")")"
-    done
-    echo "TEST_CONTRACTS_PREBUILT=1"
-  } >>"$NEXTEST_ENV"
-fi
+export NEAR_SANDBOX_RPC_URL="$(cat "$(addr_file 0)")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)